docs/tools/erc725js/methods/Encoding.md

```diff
@@ -0,0 +1,507 @@
+  schemas,
+);
+/**
+{
+  keys: ['0x0fb367364e1852abc5f20000bedbedbedbedbedbedbedbedbedbedbedbedbedb'],
+  values: ['0xcafecafecafecafecafecafecafecafecafecafe]
+}
+*/
+
+const schemas = [
+  {
+    name: 'DynamicKey:<bytes4>:<string>',
+    key: '0xForDynamicKeysThisFieldIsIrrelevantAndWillBeOverwriten',
+    keyType: 'Mapping',
+    valueType: 'bytes',
+    valueContent: 'Address',
+  },
+];
+
+myErc725.encodeData(
+  [
+    {
+      keyName: 'DynamicKey:<bytes4>:<string>',
+      dynamicKeyParts: ['0x11223344', 'Summer'],
+      value: '0xcafecafecafecafecafecafecafecafecafecafe',
+    },
+  ],
+  schemas,
+);
+/**
+{
+  keys: ['0x0fb367364e1852abc5f2000078c964cd805233eb39f2db152340079088809725'],
+  values: ['0xcafecafecafecafecafecafecafecafecafecafe']
+}
+*/
+```
+
+</details>
+
+<details>
+    <summary>Encode multiple keys at once</summary>
+
+```javascript title="Encode multiple keys at once"
+myErc725.encodeData([
+  {
+    keyName: 'LSP3Profile',
+    value: {
+      verification: {
+        method: 'keccak256(utf8)',
+        data: '0x820464ddfac1bec070cc14a8daf04129871d458f2ca94368aae8391311af6361',
+      },
+      url: 'ipfs://QmYr1VJLwerg6pEoscdhVGugo39pa6rycEZLjtRPDfW84UAx',
+    },
+  },
+  {
+    keyName: 'LSP12IssuedAssets[]',
+    value: [
+      '0xD94353D9B005B3c0A9Da169b768a31C57844e490',
+      '0xDaea594E385Fc724449E3118B2Db7E86dFBa1826',
+    ],
+  },
+  {
+    keyName: 'LSP1UniversalReceiverDelegate',
+    value: '0x1183790f29BE3cDfD0A102862fEA1a4a30b3AdAb',
+  },
+]);
+/**
+{
+  keys: [
+    '0x5ef83ad9559033e6e941db7d7c495acdce616347d28e90c7ce47cbfcfcad3bc5',
+    '0x7c8c3416d6cda87cd42c71ea1843df28ac4850354f988d55ee2eaa47b6dc05cd',
+    '0x7c8c3416d6cda87cd42c71ea1843df2800000000000000000000000000000000',
+    '0x7c8c3416d6cda87cd42c71ea1843df2800000000000000000000000000000001',
+    '0x0cfc51aec37c55a4d0b1a65c6255c4bf2fbdf6277f3cc0730c45b828b6db8b47',
+  ],
+  values: [
+    '0x6f357c6a820464ddfac1bec070cc14a8daf04129871d458f2ca94368aae8391311af6361696670733a2f2f516d597231564a4c776572673670456f73636468564775676f3339706136727963455a4c6a7452504466573834554178',
+    '0x00000000000000000000000000000002',
+    '0xd94353d9b005b3c0a9da169b768a31c57844e490',
+    '0xdaea594e385fc724449e3118b2db7e86dfba1826',
+    '0x1183790f29be3cdfd0a102862fea1a4a30b3adab',
+  ],
+}
+*/
+```
+
+</details>
+
+<details>
+    <summary>Encode array length</summary>
+
+If the key is of type Array and you pass an integer as a value (for instance, the array length), it will be encoded accordingly.
+
+```javascript title="Encode the length of an array"
+myErc725.encodeData([
+  {
+    keyName: 'LSP3IssuedAssets[]',
+    value: 5,
+  },
+]);
+/**
+{
+  keys: [
+    '0x3a47ab5bd3a594c3a8995f8fa58d0876c96819ca4516bd76100c92462f2f9dc0',
+  ],
+  values: ['0x00000000000000000000000000000005'],
+}
+*/
+```
+
+</details>
+
+<details>
+    <summary>Encode a subset of array elements</summary>
+
+```javascript title="Encode a subset of array elements"
```

Comment:
FMI so that I understant well, 
1.In this example, there are already 21 addresses stored in the permission array?

2.What are the three keys and values corresponding too in the result?